spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/StreamTemplate.java

```diff
@@ -155,6 +162,16 @@ public void scaleApplicationInstances(String streamName,
 		restTemplate.postForObject(url, properties, Object.class);
 	}
 
+	@Override
+	public String streamExecutionLog(String streamName) {
+		return restTemplate.getForObject(logsLink.expand(streamName, null).getHref(), String.class);
```

Comment:
Have you tried this with an existing stream with applications deployed? I wonder if we can use the explicit `streams/logs/{streamName}` endpoint for this.